src/librustc_codegen_ironox/builder.rs

```diff
@@ -109,374 +119,384 @@ impl BuilderMethods<'a, 'tcx> for Builder<'a, 'll, 'tcx> {
     fn build_sibling_block<'b>(&self, name: &'b str) -> Self {
         Builder::new_block(self.cx, self.llfn(), name)
     }
+
     fn cx(&self) -> &CodegenCx<'ll, 'tcx> {
         &self.cx
     }
 
-    fn llfn(&self) -> &'ll Value {
-        &Value {}
+    fn llfn(&self) -> Value {
+        unimplemented!("llfn");
     }
 
-    fn llbb(&self) -> &'ll BasicBlock {
-        &BasicBlock {}
+    fn llbb(&self) -> BasicBlock {
+        unimplemented!("llbb");
     }
 
     fn count_insn(&self, category: &str) {
         unimplemented!("count_insn");
     }
 
-    fn set_value_name(&mut self, value: <Self::CodegenCx as BackendTypes>::Value, name: &str) {
-        unimplemented!("set_value_name");
+    fn set_value_name(&mut self, value: Value, name: &str) {
+        // FIXME: rename the value
     }
 
-    fn position_at_end(&mut self, llbb: <Self::CodegenCx as BackendTypes>::BasicBlock) {
-        unimplemented!("position_at_end");
+    fn position_at_end(&mut self, llbb: BasicBlock) {
```

Comment:
Not yet, no. I'll add its implementation in my next PR. 

Some of these changes may not make much sense, because I'm trying to keep these PRs as small as I can. I should've leave this one `unimplemented`.